Adding a message to Publishing Models (#3043)
fishtown-analytics/docs.getdbt.com: website/docs/docs/collaborate/publish/about-publishing-models.md

---
title: "About publishing models"
id: about-publishing-models
description: "Information about new features related to publishing models"
---

:::info

> Publishing models contains features new in dbt Core v1.5, currently in beta. If you do not see the documentation related to publishing models in this directory, please change the version of the docs to v1.5 or higher from the dropdown menu at the top of this site.

Reviewer: Can we provide a link with the URL and version specified instead of describing it?
```suggestion
Publishing models contains features new in dbt Core v1.5, currently in beta. To see the documentation related to publishing models, make sure to [select v1.5 or higher](/docs/collaborate/publish/about-publishing-models?version=1.5).
```
